backend/app/services/knowledge/orchestrator.py

```
@@ -0,0 +1,900 @@
+                f"Missing embedding model_name in KB {knowledge_base.id}"
+            )
+            return
+
+        # Determine index owner
+        if knowledge_base.namespace == "default":
+            index_owner_user_id = user.id
+        else:
+            index_owner_user_id = knowledge_base.user_id
+
+        # Get summary_enabled flag
+        summary_enabled = spec.get("summaryEnabled", False)
```

Ruff (0.14.14)

[error] Local variable `summary_enabled` is assigned to but never used

Remove assignment to unused variable `summary_enabled`

(F841)